### ISSUE TITLE
disable color output in dumb terminal

### ISSUE THREAD

**Issue by bbreinbauer** [licheng/gccfilter#5]
I'm using gccfilter in cmake based project and use vim as editor. It would be nice if gccfilter would skip the color ouput if a dumb terminal is detected (:make in vim).

**Comment by bbreinbauer** [comment]
I'm using the following code (modified from colorgcc) which seems to work. Perl is not really my programming language...
```diff
@@ -408,6 +408,10 @@
 waitpid($pid, 0);
 my $gcc_exit_code = $? >> 8;
 my @gcc_stderr = <CHILD_ERR>;
+if (! -t STDOUT)
+{
+    $opt_colorize = 0;
+}

 # main parser
 for (@gcc_stderr){
```

**Comment by licheng** [comment]
@bbreinbauer I merged your patch in commit da68f5f, thank you ;)
